### ISSUE TITLE
Source code?

### ISSUE THREAD

**Issue by jeremyredhead** [airsdk/airsdkmanager-releases#7]
I don't understand, where is the source code ? 

can i have It ? 
please

**Comment by ajwfrost** [comment]
There's no source code here, this is just a repository to host the releases.
Curious why you would want the source code?
thanks

**Comment by jeremyredhead** [comment]
b/.c it'sd yummmu yummy 
for my
 tummy 

i will Eat it
the source code,,
 i will     (i WILL) 
Consuime its forbidden Knowledges..!
to Asecnd to= Greater powers.....

**Comment by jeremyredhead** [comment]
y'all should oPen up 
the 
Sourvc Eopen Sourc open the Code up the UYpon the

the. 
i wann Crack! (krrrack!) one open lieka Cold One. 

and by Cold One,i mean. 

ＡＩＲＳＤＫ

**Comment by Liushui-Miaomiao** [comment]
Hello, how can I localize this program? Is there a repository that can do this?

**Comment by ajwfrost** [comment]
If you look at the installed application under folder "assets/lang", you'll see a json file `en.json` which has all the localisable strings in it..  if you wanted to localise that, please feel free - save it in that folder with the appropriate language code (from the Capabilities class - https://airsdk.dev/reference/actionscript/3.0/flash/system/Capabilities.html#language)

If you're on macOS, that might actually break the app (code-signing...) but feel free to send us the file and we can add it to the installer!

thanks

**Comment by Liushui-Miaomiao** [comment]
> If you look at the installed application under folder "assets/lang", you'll see a json file `en.json` which has all the localisable strings in it.. if you wanted to localise that, please feel free - save it in that folder with the appropriate language code (from the Capabilities class - https://airsdk.dev/reference/actionscript/3.0/flash/system/Capabilities.html#language)
> 
> If you're on macOS, that might actually break the app (code-signing...) but feel free to send us the file and we can add it to the installer!
> 
> thanks

Thank you very much. Would you consider accepting these localized resources?

**Comment by ajwfrost** [comment]
> Would you consider accepting these localized resources?

I think we'd appreciate any inputs like this! Assuming no objections from @marchbold ?

**Comment by Liushui-Miaomiao** [comment]
> > Would you consider accepting these localized resources?
> 
> I think we'd appreciate any inputs like this! Assuming no objections from @marchbold ?

Thanks!

**Comment by marchbold** [comment]
Yes please! As Andrew said if you can have a look at that json file and then create a copy of it with a localised version that would be very helpful! You can send it through to me at airnativeextensions @ distriqt.com and I'll get it integrated to the next release.
